server/survey.go

```diff
@@ -82,22 +84,26 @@ func (p *Plugin) checkForNextSurvey(now time.Time) bool {
 	nextSurvey = &surveyState{
 		ServerVersion: p.serverVersion,
 		CreateAt:      now,
-		StartAt:       now.Add(TIME_UNTIL_SURVEY),
+		StartAt:       now.Add(TimeUntilSurvey),
 	}
 
 	p.API.LogInfo(fmt.Sprintf("Scheduling next survey for %s", nextSurvey.StartAt.Format("Jan 2, 2006")))
 
-	if err := p.KVSet(fmt.Sprintf(SURVEY_KEY, p.serverVersion), nextSurvey); err != nil {
+	if errSchedule := p.KVSet(fmt.Sprintf(SurveyKey, p.serverVersion), nextSurvey); errSchedule != nil {
 		p.API.LogError("Failed to schedule next survey", "err", err)
 		return false
 	}
 
-	if sent, err := p.sendAdminNotices(now, nextSurvey); err != nil {
+	sent, errNotice := p.sendAdminNotices(now, nextSurvey)
+	if errNotice != nil {
 		p.API.LogError("Failed to send notification of next survey to admins", "err", err)
 		return false
-	} else if !sent {
+	}
+
+	switch sent {
+	case false:
 		p.API.LogInfo("Not sending notification of next survey to admins since they already received one recently")
-	} else {
+	default:
```

Comment:
Is this change really needed?

Reply:
I'm not sure about this, but when running golangci-lint, linter say if the *if-else chain* must change with *switch case*